benchmarks/Swaggatherer/Program.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Swaggatherer
```

Comment:
Swagger + Gatherer = 😆

Reply:
Now I wonder which judge is the mean one and which judge is the nice one